integrations/access/accessrequest/app.go

```diff
@@ -262,6 +267,11 @@ func (a *App) onPendingRequest(ctx context.Context, req types.AccessRequest) err
 		} else {
 			log.Warning("No channel to post")
 		}
+
+		// Try to approve the request if user is currently on-call
+		if err := a.tryApproveRequest(ctx, reqID, req); err != nil {
+			log.Warningf("Failed to auto approve request: %v", err)
+		}
```

Comment:
I think we should run this only if `teleportUser` is not empty. This will reduce the noise against the Teleport cluster from non-configured plugins.

Reply:
The `teleportUser` is not explicitly required for "self-hosted" plugins, and I don't think we should make it explicitly required since that will add more complexity to the configuration.

The function should log in debug level in most cases and only log a warning if the code execution reaches the point where it attempts to submit an access review.